application.py

```diff
@@ -10,36 +11,36 @@
 from structlog.stdlib import LoggerFactory, add_log_level
 from structlog.threadlocal import wrap_dict
 
-EQ_LOG_LEVEL = os.getenv('EQ_LOG_LEVEL', 'INFO')
 EQ_WERKZEUG_LOG_LEVEL = os.getenv('EQ_WERKZEUG_LOG_LEVEL', 'INFO')
 EQ_DEVELOPER_LOGGING = os.getenv('EQ_DEVELOPER_LOGGING', 'False').upper() == 'TRUE'
 
 
 def configure_logging():
-    # set up some sane logging, as opposed to what flask does by default
-    log_format = "%(message)s"
-    levels = {
-        'CRITICAL': logging.CRITICAL,
-        'ERROR': logging.ERROR,
-        'WARNING': logging.WARNING,
-        'INFO': logging.INFO,
-        'DEBUG': logging.DEBUG,
-    }
-    handler = logging.StreamHandler()
+    log_level = logging.INFO
+    if os.getenv('FLASK_ENV') == 'development':
+        log_level = logging.DEBUG
+
+    info_handler = logging.StreamHandler(sys.stdout)
+    info_handler.setLevel(log_level)
+    info_handler.addFilter(lambda record: record.levelno <= logging.WARNING)
```

Comment:
Given the `info_handler` will handle `WARNING` and `DEBUG` it might be clearer to name the handlers `log_handler` and `error_log_handler`